src/randoop/main/GenInputsAbstract.java

```diff
@@ -409,17 +373,12 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean dont_output_tests = false;
   
   /**
-   * Output sequences even if they do not complete execution.
-   * 
-   *  Randoop's default behavior is to output only tests consisting of
-   *  method call sequences that execute every statement, rather than throwing
-   *  an exception or failing a contract check before the last statement.
+   * Indicate which classes that any generated test must include. Tests that
+   * do not have classes matching the regular expression are not written to 
+   * the test suites. 
    */
-  @Option("Output sequences even if they do not complete execution")
-  public static boolean output_nonexec = false;
-  
-  @Option("specifies regex of classes that must be in any regression tests")
-  public static Pattern test_classes = null;
+  @Option("Regular expression for class names that must occur in generated tests")
```

Comment:
The predicate actually was only returning false if there were variables and none of them matched the pattern, but I changed it so that it also returns false when there are no variables.

But otherwise the description is correct. The predicate loops through the variables of a test sequence until it finds one that matches the regex, and only returns true if one is found.